header-footer-grid/assets/scss/frontend/components/_palette-switch.scss

```diff
@@ -0,0 +1,30 @@
+.toggle-palette {
+
+  a {
+	text-decoration: none;
+	font-size: 0.8em;
+	line-height: 1.2em;
+	letter-spacing: 0;
+	span {
+	  display: inline-flex;
+	  &.icon {
+		vertical-align: middle;
+	  }
+	  &.label {
+		display: inline-flex;
+		font-size: small;
+		vertical-align: middle;
+		color: var(--nv-primary-accent);
```

Comment:
please validate with @JohnPixle the color of the icon/label

Reply:
@preda-bogdan @selul the color is correct as primary accent, thanks!